python-avd/pyavd/__init__.py

```diff
@@ -28,3 +29,5 @@
     "validate_structured_config",
     "ValidationResult",
 ]
+
+check_templates()
```

Comment:
As mentioned I think we should move this into init of the templar, so we avoid doing this on every instance if not needed.

Reply:
yeah this will go once it is moved to verify_req